Bottom: a51fc86c8b3ed1bce6877badf02b729eee8de9a6
Top:    93993b952dd7409cf7520a2df27a564d07c761a4
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-03 13:15:00 +0800

Refresh of vhost-packed-ring-support

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 215d72b..0f3f07c 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2095,7 +2095,8 @@ static int get_indirect_packed(struct vhost_virtqueue *vq,
 			 * to come before any input descriptors.
 			 */
 			if (unlikely(*in_num)) {
-				vq_err(vq, "Indirect descriptor has out after in: idx %d\n", i);
+				vq_err(vq, "Indirect descriptor has out after in: idx %d\n",
+				       i);
 				return -EINVAL;
 			}
 			*out_num += ret;
@@ -2208,7 +2209,8 @@ static int vhost_get_vq_desc_packed(struct vhost_virtqueue *vq,
 						  log_num, &desc);
 			if (unlikely(ret < 0)) {
 				if (ret != -EAGAIN)
-					vq_err(vq, "Failure detected in indirect descriptor at idx %d\n", i);
+					vq_err(vq, "Failure detected in indirect descriptor at idx %d\n",
+					       i);
 				return ret;
 			}
 			goto next;
diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index 4c7c32d..839ae7e 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -162,10 +162,10 @@ struct vhost_memory {
 
 /* Base value where queue looks for used descriptors */
 #define VHOST_SET_VRING_USED_BASE _IOW(VHOST_VIRTIO, 0x25,	\
-                                  struct vhost_vring_state)
+				  struct vhost_vring_state)
 /* Get accessor: reads index, writes value in num */
 #define VHOST_GET_VRING_USED_BASE _IOWR(VHOST_VIRTIO, 0x26,	\
-                                  struct vhost_vring_state)
+				  struct vhost_vring_state)
 
 /* VHOST_NET specific defines */
